packages/eslint-plugin-next/lib/rules/no-document-import-in-page.js

```diff
@@ -18,8 +20,8 @@ module.exports = {
 
         if (
           !page ||
-          page.startsWith('/_document') ||
-          page.startsWith('\\_document')
+          page.startsWith(`${path.sep}_document`) ||
+          page.startsWith(`${path.posix.sep}_document`)
```

Comment:
Did you test this on Windows?

For what I can tell, this will behave identical to the existing code.

https://nodejs.org/api/path.html#path_path_sep

Reply:
Didn't test it on Windows (don't have a VM I can use unfortunately), but I didn't submit this change to fix the particular issue of it it still failing in some instances. Just thought it would make more sense to use `path.sep` as @kimbaudi suggested even though it'll likely behave the exact same way.

Reply:
sorry, i'm the one that was confused with `path.sep` and `path.posix.sep`. I just tested on Windows and `path.sep` is `\\` and `path.posix.sep` is `/`. and yes, this will behave same as existing code.

Reply:
Just had a build on Windows fail in CI (GitHub actions) due to this exact rule:

```
Failed to compile.

./pages/_document.js
3:1  Error: next/document should not be imported outside of pages/_document.js. See https://nextjs.org/docs/messages/no-document-import-in-page.  @next/next/no-document-import-in-page
```

Assuming it's because of the `./` relative path?

Reply:
Just looked across the code, actually it seems to check out. So, now I'm not entirely sure what's gone wrong...